test/e2e/user_management_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"io/ioutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
this `import` statement shouldn't have been moved in the first section (usually reserved for standard packages)

Reply:
yeah, the editor was trying to be helpful and i didn't catch it before pushing! will fix this!